Microsoft.Toolkit.Uwp.SampleApp/SamplePages/WeatherLiveTileAndToast/WeatherLiveTileAndToastPage.xaml.cs

```diff
@@ -119,10 +116,9 @@ private static bool IsAdaptiveToastSupported()
         {
             switch (AnalyticsInfo.VersionInfo.DeviceFamily)
             {
-                // Desktop and Mobile started supporting adaptive toasts in API contract 3 (Anniversary Update)
                 case "Windows.Mobile":
```

Comment:
Since Windows 10 Mobile is stuck on build 15063, wouldn't this case never be used? 🤔

Reply:
Notification SDKs still target 10240 as the min build (we're not increasing the min target of the Notification SDK since this is the official way to interact with notifications).

Reply:
@andrewleader should we keep the version check right bellow this then? I forgot about this.

Reply:
I'll keep this change since the sample app doesn't go all the way down to RS1, but I'll revert the change on the binding file so whenever someone uses the sample app they see that this is a good pattern to follow, since this lib goes all the way down.

Reply:
Sounds perfect!

Reply:
@andrewleader I'm a bit confused, I mean even though the Notification SDK supports down to 10240, the sample app itself is locked to a minimum of 17763 now, so it'll never actually run on mobile, right? I'm not sure I understand when that line could ever be reached 🤔

I mean technically this would've been the case already since the minimum was 16299 before this PR, which was still outside of the reach for Windows 10 Mobile. Or I might just be missing something obvious here 😅

Reply:
On the C# side, its just to match more closely the binding file. Ideally they would just be the same. Still, I'm trying to remove as much as possible of conditional runtime paths that uses ApiInformation, so that's why I removed it from the C# file. I could also remove the check for mobile, as it will never hit. I'll do that as well.

Reply:
Ha, perfect! And thanks for the extra info! 😊